### PR TITLE
Testing branch

### PR DESCRIPTION
**Pull request by AmartC**

### CODE REVIEW

**Comment by sgober** [comment]
Linting fails

**Comment by sgober** [comment]
Maybe elaborate the dashboard tests a little bit more.  For instance adding a clicking test for the icon view vs the full nav view or testing to make sure all the nav list items are there.

File: src/__mocks__/providerMocks.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+const providerMockA = {
+  id: 1,
+  name: "Fitbit",
+};
+
+const providerMockB = {
+  id: 2,
+  name: "Massachusetts General Hospital",
```
Comment (sgober) [inline src/__mocks__/providerMocks.js]: Use single quotes here

File: src/__tests__/containers/dashboard/Dashboard.test.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      username: 'abcd',
+      isAuthenticated: 'true'
+    },
+    profiles: {
+      activeProfile: profileMockA,
+    }
+  };
+
+  const props = {
+    authUser: "abcd",
```
Comment (sgober) [inline src/__tests__/containers/dashboard/Dashboard.test.js]: Also here

**Comment by sgober** [comment]
Added some syntax comments.  I would also add a test that the click handling of opening and closing the drawer adjusts the display correctly.

**Comment by noranda** [comment]
Fixed and added tests for `Profiles` and `Dashboard` containers.

**Pull request merged**